sample-guestbook-app-ocm-ns/channel.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: apps.open-cluster-management.io/v1
+kind: Channel
+metadata:
+  name: git-helm-ch
+  namespace: gb-app-latest-ns
+spec:
+  pathname: 'https://github.com/rokej/testrepo.git'
```

Comment:
@richiepiya can we have the resources from https://github.com/rokej/testrepo.git moved to this repo ( in case Roke updates the content, we don't want to depend on him )
You can create a new folder for the resources, as is done for the other samples ( mortgageds, dc ) and put the chart there

Reply:
moved the contents of that repo to a folder called resources-roke